RZTouchID/RZTouchID.m

```diff
@@ -65,16 +63,27 @@ + (BOOL)touchIDAvailable
     return available;
 }
 
-- (instancetype)initWithKeychainServicePrefix:(NSString *)servicePrefix
+- (BOOL)touchIDAvailableForIdentifier:(NSString *)userID
+{
+    return ( userID != nil && userID.length > 0 && [self.class touchIDAvailable] && [self.delegate touchIDAvailableForUserID:userID] );
+}
+
+- (instancetype)initWithKeychainServicePrefix:(NSString *)servicePrefix touchIDMode:(RZTouchIDMode)touchIDMode
 {
     self = [super init];
     if ( self ) {
         self.keychainServicePrefix = servicePrefix;
         self.completionQueue = dispatch_get_main_queue();
+        self.touchIDMode = touchIDMode;
     }
     return self;
 }
 
+- (void)setLocalizedFallbackTitle:(NSString *)localizedFallbackTitle
+{
+    _localizedFallbackTitle = localizedFallbackTitle;
+}
```

Comment:
I don't think this method needs to be here. It will be auto synthesized since it's declared as a property, and it's declared in the header so it will be public. If you leave it here though, make sure to do `_localizedFallbackTitle = [localizedFallbackTile copy];` to enforce copy semantics